doc/source/Getting_started/Troubleshooting.rst

```diff
@@ -0,0 +1,184 @@
+
+
+Another option to install PyAedt from the wheelhouse is to download the following file
+:download:`PyAEDT Installer Python file <../Resources/PyAEDTInstallerFromDesktop.py>`.
+Run this script directly from AEDT and pass the wheelhouse file name as an argument.
+
+
+
+
+Run PyAEDT
+~~~~~~~~~~
+
+COM vs GRPC
+-----------
+Prior to the 2022R2 release CPython automation in AEDT used
+`COM <https://learn.microsoft.com/en-us/windows/win32/com/com-objects-and-interfaces>`_  which
+requires all interfaces to be registered in the Windows Registry.
+Communication between Python and the AEDT API were translated through an intermediate layer using
+`pywin32 <https://github.com/mhammond/pywin32>`_. The recent addition of
+`PythonNET <https://pythonnet.github.io/pythonnet/>`_ improves
+cross-platform support and the COM interface is replaced by `GRPC <https://grpc.io/>`_,
+for 2022R2 and later.
+
+
+.. list-table:: GRPC Compatibility Table
+   :widths: 25 25 75 75
+   :header-rows: 1
+
+   * -
+     - <2022R2
+     - 2022R2
+     - >2022R2
+   * - AEDT
+     - Only PythonNET
+     - GRPC Beta
+     - GRPC Available
+   * - AEDT
+     - Only PythonNET
+     - PythonNET default.
+       GRPC: pyaedt.settings.use_grpc_api = True
+     - GRPC default
+       PythonNET: pyaedt.settings.use_grpc_api = False
+
+The options shown here apply only to the Windows platform.
+On Linux, the Python interface to AEDT uses GRPC for all versions.
+
+
+Check the Electronics Desktop API configuration
+-----------------------------------------------------
+To start the Electronics Desktop server in GRPC mode use the following syntax:
+
+.. code:: bat
+   :caption: Windows
+
+   path\to\AnsysEM\v231\Win64\ansysedt.exe -grpcsrv 50001
+
+.. code:: console
+   :caption: Linux
+
+   path\to\AnsysEM\v231\Lin64\ansysedt -grpcsrv 50352
+
+The server port number is used by AEDT to listen and receive
+commands from PyAEDT client instances. This configuration
+supports multiple sessions of AEDT running on a single server
+and listening on the same port.
+
+Check the GRPC Interface
+------------------------
+The native Electronics Desktop API can be used to launch
+Electronics Desktop from the command line.
+This can be done even without PyAEDT to check that everything is set up correctly
+and all environment
+variables have been defined.
+
+.. code:: python
+
+    import sys
+    sys.path.append(r"ANSYSEM_ROOT231\PythonFiles\DesktopPlugin")
+    import ScriptEnv
+    print(dir())
+    ScriptEnv.Initialize("", False, "", 50051)
+    print(dir())
+
+
+
+Failures in connecting to GRPC API
+----------------------------------
+On Linux, it may happens that PyAEDT fails to initialize a new session of Electronics Desktop
+or to connect to an existing one.
+This may be due to:
+ - Firewall
+ - Proxy
+ - Permissions
+ - License
+ - Scheduler used to launch AEDT like LSF
+
+In case of issues with proxy, you may try the following environment variable:
```

Comment:
Do you mean the server is running a proxy server?
```suggestion
In case of issues due to use of a proxy server, you may set the following environment variable:
```